src/astarte_message_hub.rs

```diff
@@ -184,8 +374,9 @@ impl<T: Clone + AstartePublisher + AstarteSubscriber + 'static>
         &self,
         request: Request<AstarteMessage>,
     ) -> Result<Response<pbjson_types::Empty>, Status> {
-        // TODO: print node id, when it's moved to the metadata
-        info!("Node Send Request");
+        if let Some(node_id) = request.extensions().get::<NodeId>() {
+            debug!("Node {node_id} Send Request");
+        }
```

Comment:
Add a method or trait to extract the node id or return the AbsentId/MissingId error